docs/articles/documentation/guides/advanced-guides/multiple-browser-windows.md

```diff
@@ -0,0 +1,136 @@
+---
+layout: docs
+title: Multiple Browser Windows
+permalink: /documentation/guides/advanced-guides/multiple-browser-windows.html
+---
+# Multiple Browser Windows
+
+The TestCafe API includes methods that open, close, and switch between browser windows. You can test websites with pop-up windows and OAuth login forms, debug complex multi-window applications, or run multiple instances of the same web app side by side.
+
+>This is a beta feature. Browser support is limited to local instances of Chrome and Firefox. Videos and screenshots of child windows cannot be captured. The available functionality is subject to further revisions. Use at your own risk.
```

Comment:
> Use at your own risk.

We were just going to say that we don't recommend production usage. There's no reason to talk about risks.